rust/processor/src/db/postgres/models/coin_models/coin_supply.rs

```diff
@@ -53,8 +53,13 @@ impl CoinSupply {
             }
 
             // Convert to TableItem model. Some fields are just placeholders
-            let (table_item_model, _) =
-                TableItem::from_write_table_item(write_table_item, 0, txn_version, 0);
+            let table_item_model = RawTableItem::postgres_table_item_from_write_item(
+                write_table_item,
+                0,
+                txn_version,
+                0,
+                txn_timestamp,
+            );
```

Comment:
nit: we use the helper `postgres_table_item_from_write_item` here and 
```
            let (raw_table_item, _current_table_item) = RawTableItem::from_write_table_item(
                write_table_item,
                0,
                txn_version,
                0,
                block_timestamp,
            );
            let table_item_model: TableItem = TableItem::from_raw(&raw_table_item);
```
in other in `delegator_balacnes.rs`. let's keep it consistent!